dcos_core_variables.tf

```diff
@@ -479,7 +479,22 @@ variable "dcos_ip_detect_contents" {
 }
 
 variable "dcos_rexray_config" {
-  default     = ""
+  default = <<EOF
+# YAML
+  rexray:
```

Comment:
did you test this? IMO this will end up in indentation issues.

The object should be defined at root level the template takes care of indentation.